extensions/Vendor/FRAUNHOFER_materials_pbr/README.md

```diff
@@ -0,0 +1,146 @@
+# FRAUNHOFER_materials_pbr
+
+## Contributors
+
+* Timo Sturm, Fraunhofer IGD, [@\_tsturm\_](https://twitter.com/\_tsturm\_)
+* Miguel Sousa, Fraunhofer IGD, [@mfportela](https://twitter.com/mfportela)
+* Maik Th&ouml;ner, Fraunhofer IGD, [@mthoener](https://twitter.com/mthoener)
+* Max Limper, Fraunhofer IGD, [@mlimper_cg](https://twitter.com/mlimper_cg)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 1.0 spec.
+
+## Overview
+
+The glTF 1.0 allows the definition of materials by instancing `techniques`. A `technique`, as defined in glTF 1.0, is a verbose description of shader parameters combined with actual shader code. Typically, shader code is engine specific and, as such, may not be used across systems which do not share the same rendering pipeline.
+
+This extension provides two new `techniques` consisting of a well defined set of parameters which are sufficient for representing a wide range of materials. These techniques are based on widely used material representations for Physically-Based Rendering (PBR) content creation pipelines.
+
+
+
+## Material models
+
+<p style="text-align:justify;">A material model defines a set of parameters used to describe a material. This extension supports two material models commonly used in PBR, namely the specular-glossiness and the metal-roughness models. An implementation of this extension should support both models.</p>
+
+### Specular - Glossiness
+
+<img src="figures/specular_glossiness_2.png" align="middle" height="220" style="display: block; margin: 0 auto; padding: 20px 0 10px 0;">
```

Comment:
Yes, we created those figures (also for the equations). We used them in the respective ACM Web3D paper, so we should double-check that it is OK to use them for non-commercial purposes. I would guess so, but I am not 100% sure, so: good point, we need to check (or simply create replacements specifically for this extension).